Makefile

```diff
@@ -51,8 +59,20 @@ production:
 
 .PHONY: build-local-image
 build-local-image:
+		$(eval export DOCKER_BUILDKIT=1)
+		$(eval branch=$(shell git rev-parse --abbrev-ref HEAD))
 		$(eval tag=dev-$(shell git rev-parse HEAD)-$(shell date '+%Y%m%d%H%M%S'))
-		docker build -t $(repository):$(tag) .
+		docker build \
+			--build-arg BUILDKIT_INLINE_CACHE=1 \
+			--cache-from $(repository):builder-master \
```

Comment:
Can we add:
--cache-from $(repository):builder-$(branch) \
and
--cache-from $(repository):master \
?

Reply:
agreed